/obstacle_trajectories_node:
  ros__parameters:
    map_frame: map
    planner_frame: planner
    trajectory_topic: trajectory
    planner_frame: planner
    # do_visualise: True
